### DISCUSSION TITLE
CustomRenderer - Draw method with html?

### DISCUSSION THREAD

**Discussion by antoniobeltran** [glideapps/glide-data-grid#616] [Q&A] [answered]
I've been trying to see if I can use existing HTML and render it inside the draw so I can render html in the cell, but it's a canvas, so looking to see if I can hack it in. Do you have any recommendations, or should I ditch this and just use ctx to draw?

**Comment by jassmith** [comment, marked as answer]
No way to do that efficiently. If you need HTML I suggest using something else.
